spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
```

Comment:
> why your the other comment doesn't follow this thread

I responded directly from Gmail.

I don't think that's a good example because it's an edge case where Rails doesn't do what's expected, but will now continue to do exactly what we expect. There should be no reason that the inflection need be changed in the future, and if for some reason it does then said person should probably ask themselves why the correction was there in the first place :)

Inflections are odd because Rails specifically states that not every single word works as expected and they won't accept pull requests for a never-ending number of exceptions. You're directed to do exactly what @bdunne's solution does for your own uses.